packages/teams-js/test/public/call.spec.ts

```diff
@@ -22,13 +22,17 @@ describe('call', () => {
     }
   });
 
-  it('should not allow calls before initialization', () => {
-    return expect(call.startCall(mockStartCallParams)).rejects.toThrowError('The library has not yet been initialized');
```

Comment:
![image](https://user-images.githubusercontent.com/88564543/151897096-9d20e95e-4e41-406c-b8a2-874395222c7e.png)

The above code looks correct to me. Attached is the pic from official jest doc. 
Are we changing it for code readability?

Reply:
Mentioned this on the `communication.spec.ts` PR, but I thought that `.rejects` returned the rejected value based on https://jestjs.io/docs/asynchronous#resolves--rejects, so I thought it needed to be treated as an object rather than a function. These tests will work either way, so we should just pick an approach and be consistent with it. I don't have a strong preference here.

Reply:
As mentioned offline, we're looking into getting consistent guidance here. In the meantime, can you confirm that if the test fails, it reports failure correctly (e.g., showing a red X in VS Code with the Jest plugin, reporting failed test in the command line summary, etc.)?

Reply:
Sorry for the delay, but I have confirmed that if the test fails it is reported correctly in both implementations.

Reply:
**We have two different ways to go for it but there is a difference as some functions don't return a promise.**

**Function which returns Promise the code we can use either as below:**

`try {
      await call.startCall(mockStartCallParams);
    } catch (e) {
      expect(e).toEqual(new Error('The library has not yet been initialized'));
    }`
or 

`await expect(call.startCall(mockStartCallParams)).rejects.toThrowError('The library has not yet been initialized');`

**Function which does not return Promise we need to use the try method one as using await will make no difference.**
This is the function which I've been working on it's in **_public/teamsAPI.spec.ts_**
`     try {
        teamsCore.registerOnLoadHandler(() => {
          return false;
        });
      } catch (e) {
        expect(e).toEqual(new Error('The library has not yet been initialized'));
      }`

For code consistency I'd recommend **try** method.

Reply:
This is good discussion! Erin recently published some internal docs about how we want to test this case and why, and it matches what we have here: we prefer async/await to return "because it more closely resembles how app developers would call the function".